nbs/nbdev_comments.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>Can you make these refer to comment instead of flag as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/418/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>